Gordon360/Models/ViewModels/CountryViewModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Gordon360.Models.CCT;
+
+namespace Gordon360.Models.ViewModels
+{
+    public class CountryViewModel
+    {
+        public string CTY { get; set; }
+        public string COUNTRY { get; set; }
+        public static implicit operator CountryViewModel(Countries c)
+        {
+            CountryViewModel vm = new CountryViewModel
+            {
+                CTY = c.CTY,
+                COUNTRY = c.COUNTRY,
+            };
+            return vm;
+        }
+    }
```

Comment:
Did you create this class? I'm not sure what the purpose of the view model is here, since it mirrors exactly the structure of the db model it's implementing. Why not just use the `Countries` class directly?